### PR TITLE
Specify the default source energy distribution in docs

### PR DESCRIPTION
**Pull request by smharper**
Just filling a small gap in our docs.

### CODE REVIEW

File: docs/source/usersguide/input.rst [hunk 1]
```diff
@@ -528,6 +528,8 @@ attributes/sub-elements:
     sub-elements/attributes are those of a univariate probability distribution
     (see the description in :ref:`univariate`).
 
+    *Default*: Watt spectrum with :math:`a` = 0.988 MeV and :math:`b` = 2.249 MeV
```
Comment (smharper) [inline docs/source/usersguide/input.rst]: Whoops, so it is.  Relatedly, is it possible to make one of these `*Default*` statements with a line-break in it?  When I try with a line-break, it makes the statement bold.

Comment (samuelshaner) [inline docs/source/usersguide/input.rst]: I think so. You can look at the cell temperature sub-element for an example.

Comment (smharper) [inline docs/source/usersguide/input.rst]: Figured it out.  Thanks for the pointer

**Comment by samuelshaner** [comment]
Looks good!

**Pull request merged**